app/components/works/subtypes_component.html.erb

```diff
@@ -39,7 +39,7 @@
     </div>
 
     <% unless mixed_material_type? %>
-      <a href="#" class="more-options collapsed" data-action="more-types#toggleMoreTypes" data-more-types-target="moreTypesLink">
+      <a href="#" class="more-options collapsed" data-action="more-types#toggleMoreTypes" data-more-types-target="moreTypesLink" aria-expanded="false" role="button">
```

Comment:
Does this need a `aria-controls` attribute as well?

Reply:
Added.